src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,89 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  items: string[];
```

Comment:
Items prop is missing in the task description
![image](https://github.com/mate-academy/react_pagination/assets/96409563/175a783e-4373-490c-aa66-7ab8e5f683cf)

Reply:
Don't understand, if I need items in Pagination component, how I could take them in different way?

Reply:
Pagination should separate from list. List of items move to another component